### PR TITLE
SWARM-986: Non main() way to create Keycloak Security Constraints settings

### PR DESCRIPTION
**Pull request by emag**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

Motivation
----------
Currently Keyclaok Fraction Security Constraints settings requires a main().
It would be good if we could support it as Configurable.

Modifications
-------------
SecuredArchivePreparer set up the security constraints when provided 'swarm.keycloak.security-constraints'.

Result
------
You can now set Keycloak Security Constrains with project-stages.yml, no main().

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]

**Comment by kenfinnigan** [comment]
retest this please

File: fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java [hunk 1]
```diff
@@ -100,6 +128,9 @@ private Asset createAsset(InputStream in) {
     }
 
     @Configurable("swarm.keycloak.json.path")
-    private String keycloakJsonPath;
+    String keycloakJsonPath;
+
+    @Configurable("swarm.keycloak.security-constraints")
```
Comment (kenfinnigan) [inline fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java]: Not sure, but wondering if `security-constraints` should be `security.constraints`?

**Comment by kenfinnigan** [comment]
@emag Could you update PR based on current code style? Checkstyle is flagging a couple of issues.

File: fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java (hunk 1 above)
Comment (emag) [reply fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java]: Yes, I don't have any particular reason to security'-'constraints. I'll change it to dot style.

**Comment by emag** [comment]
OK, I've just updated to latest/fixed some styles and Checkstyle doesn't complain any more. I'll resent this p-r after passing tests in my local.

**Comment by kenfinnigan** [comment]
Great! Thanks

**Pull request merged**